pytket_qir/qir.py

```diff
@@ -400,11 +482,78 @@ def condition_zero_block():
                 one=lambda: condition_one_block(),
                 zero=lambda: condition_zero_block(),
             )
+        elif isinstance(op, WASMOp):
+            cregs = _retrieve_registers(circ.bits, BitRegister)
+            args = command.args
+
+            inputs: List[str] = []
+            outputs: List[str] = []
+            for reglist, sizes in [
+                (inputs, op.input_widths),
+                (outputs, op.output_widths),
+            ]:
+                for in_width in sizes:
+                    assert in_width >= 0
+                    wasm_bits = args[:in_width]
+                    args = args[in_width:]
+                    regname = wasm_bits[0].reg_name
```

Comment:
Actually there is no need for this assertion because the corner case is that if `size` is empty. In that case, this loop is not executed and `reglist` (therefore `inputs`) is empty. This is handled later in the `try/except` block here:

```python
try:
                bit_reg = circ.get_c_register(inputs[0])
                input_type_list = [types.Int(wasm_int_size)]
 except IndexError:
                input_type_list = []

```